ivadomed/loader/loader.py

```diff
@@ -327,17 +356,18 @@ def get_pair_slice(self, slice_index, gt_type="segmentation"):
             gt_slices = []
             for gt_obj in gt_dataobj:
                 if gt_type == "segmentation":
-                    gt_slices.append(np.asarray(gt_obj[..., slice_index],
-                                                dtype=np.float32))
+                    if not isinstance(gt_obj, list):  # annotation from only one rater
+                        gt_slices.append(np.asarray(gt_obj[..., slice_index],
+                                                    dtype=np.float32))
```

Comment:
The behavior was not introduced here, but shouldn't we change dtype according to soft_gt param here?